cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -347,6 +364,55 @@ public void init() {
 
                 .routes(this::configureRoutes)
                 .javalinServlet();
+        setupQueuing();
+    }
+
+    private void setupQueuing() throws ServletException {
+        try {
+            //TODO: determine how the port is configured
+            String activeMqUrl = "tcp://" + InetAddress.getLocalHost().getHostName() + ":61616";
```

Comment:
61616 is the default ActiveMQ port, so this should be fairly safe to assume.

Reply:
But I think it would also be reasonable to only use the in-memory version of ActiveMQ in this context... but I haven't read through the rest of the code yet.

Reply:
https://activemq.apache.org/components/artemis/documentation/1.4.0/configuring-transports.html

TCP or HTTP can be used, I don't know if district T7's would allow TCP connections from the port.